### DISCUSSION TITLE
How to enable IPV6 on Windows

### DISCUSSION THREAD

**Discussion by CoenraadS** [containers/podman#22751] [Q&A]
Hello

I am currently facing an issue where my application works with Docker, but not Podman (https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/discussions/375), because it tries to connect using `::1` instead of `127.0.0.1`

I tried to search how to enable IPV6, but mostly found topics related to linux: https://developers.redhat.com/articles/2022/08/10/how-conifgure-podman-40-ipv6

How can I enable it on Windows (using linux containers) (Podman v5.0.2)?

**Comment by Diver0218** [comment]
Did you find a solution yet?
